src/Apps/Components/Layouts/index.tsx

```diff
@@ -28,7 +28,9 @@ export const LAYOUTS = {
 export type LayoutVariant = keyof typeof LAYOUTS
 
 export const Layout: FC<LayoutProps> = ({ variant = "Default", children }) => {
-  const { isFetching } = useSystemContext()
```

Comment:
No longer need this since we can just look at `match.elements` to see if we're fetching.